### DISCUSSION TITLE
non-resettable parameter

### DISCUSSION THREAD

**Discussion by lukras** [RainbowMiner/RainbowMiner#1753] [General]
where do I write the parameter for NBminer.ps1 for Autolykos2 so that after the update it is not reset and applied: --mt 6 (barks big gain) and so that the MinMemGb = 2 parameter is not reset, 3 Gb cards are mined only with the parameter 2.5 and below, you also have to change the parameters and for cngpu

[PSCustomObject] @ {MainAlgorithm = "Autolykos2";
SecondaryAlgorithm = ""; Params = "-a ergo --mt 6"; NH = $ true; MinMemGb = 2; DevFee = 2.0; Vendor = @ ("AMD", "NVIDIA"); ExtendInterval = 2; Penalty = 0; NoCPUMining = $ false} # Autolycos2 / ERGO

**Comment by RainbowMiner** [comment]
All extra command-line parameters can be set in file `.\Config\miners.config.txt` - the data records are indexed with Minername-Devicemodel. The MinMemGb cannot be set - the minimum GPU memory requirement is reduced to 2GB in release v4.7.5.1

e.g. if you had a Nvidia GTX1070, you would find the following entry in miners.config.txt. Add your extra parameters to value `Params`:
```
  "NBminer-GTX1070": [
    {
      "MainAlgorithm": "Autolykos2",
      "SecondaryAlgorithm": "",
      "Params": "--mt 6",
      "MSIAprofile": "",
      "OCprofile": "",
      "Difficulty": "",
      "Penalty": "",
      "Disable": "0",
      "ShareCheck": ""
    },
    {
      "MainAlgorithm": "BeamHash3",
      "SecondaryAlgorithm": "",
      "Params": "",
      "MSIAprofile": "",
      "OCprofile": "",
...
```
